src/v2/Components/NavBar/Menus/MobileNavMenu/LoggedInLinks.tsx

```diff
@@ -16,6 +17,39 @@ export const LoggedInLinks: React.FC<
   { error?: any } & Partial<LoggedInLinksQueryResponse>
 > = ({ error, me }) => {
   const { mediator, user } = useSystemContext()
+  const menu = {
+    title: "Account",
+    links: [
+      {
+        text: "Saves & Follows",
+        href: "/user/saves",
+      },
+      {
+        text: "Auctions",
+        href: "/user/auctions",
+      },
+      {
+        text: "Collector Profile",
+        href: "/profile/edit",
+      },
+      {
+        text: "Settings",
+        href: "/user/edit",
+      },
+      {
+        text: "Payments",
+        href: "/user/payments",
+      },
+      {
+        text: "Log out",
+        href: "#logout",
+        onClick: event => {
+          event.preventDefault()
+          mediator.trigger("auth:logout")
+        },
+      },
+    ],
+  }
```

Comment:
Why move this out of JSX?

Reply:
`<MobileSubmenuLink>` is already set up to work with this kind of structure, but it seemed cleaner and more readable to put it in a `const` rather than embedding it directly into the JSX. I tend to aim for clean JSX with references to pre-defined vars as much as possible, cos I find that JSX is already obfuscated enough without filling it up with JS structures. I'm very happy to have my mind changed on this though, if you think embedding is more readable?

Reply:
Ah I see 👍 — though I would pull it out into a real const outside of the closure here.

Reply:
Yeah, I thought that too, but I need access to the mediator to trigger a logout event, and that comes from the `useSystemContext` hook

Reply:
👍 👍